portal/src/main/webapp/js/src/cross-cancer-plotly-plots.js

```diff
@@ -515,80 +520,97 @@ var ccPlots = (function (Plotly, _, $) {
+
+                            // -- init download buttons --
+                            $("#cc_plots_svg_download").click(function() {
+                                var xmlSerializer = new XMLSerializer();
+                                var main_plots_str = xmlSerializer.serializeToString($("#cc_plots_box svg")[0]);
+                                main_plots_str = main_plots_str.substring(0, main_plots_str.length - 6);
+                                var legend_str = xmlSerializer.serializeToString($("#cc_plots_box svg")[2]);
+                                legend_str = legend_str.substring(legend_str.indexOf(">") + 1, legend_str.length);
+                                cbio.download.clientSideDownload([main_plots_str + legend_str], "cross-cancer-plots-download.svg", "application/svg+xml");
+                            });
+                            $("#cc_plots_pdf_download").click(function() {
+                                var xmlSerializer = new XMLSerializer();
+                                var main_plots_str = xmlSerializer.serializeToString($("#cc_plots_box svg")[0]);
+                                main_plots_str = main_plots_str.substring(0, main_plots_str.length - 6);
+                                var legend_str = xmlSerializer.serializeToString($("#cc_plots_box svg")[2]);
+                                legend_str = legend_str.substring(legend_str.indexOf(">") + 1, legend_str.length);
+                                var final_pdf_str = main_plots_str + legend_str;
+
+                                final_pdf_str = final_pdf_str.replace(/"/g, "'");
+                                final_pdf_str = final_pdf_str.replace("xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink'", "");
+                                final_pdf_str = final_pdf_str.replace(/text-anchor='end'/g, "");
+                                final_pdf_str = final_pdf_str.replace(/text-anchor='start'/g, "");
+                                final_pdf_str = final_pdf_str.replace(/text-anchor='middle'/g, "");
+                                final_pdf_str = final_pdf_str.replace(/text-anchor: start;/g, "");
+                                final_pdf_str = final_pdf_str.replace(/font-family: 'Open Sans',/g, "");
+                                final_pdf_str = final_pdf_str.replace(/fill: transparent;/g, "fill-opacity: 0;");
+                                final_pdf_str = final_pdf_str.replace(/'/g, "\"");
+
+                                var downloadOptions = {
+                                    filename: "cross-cancer-plots.pdf",
+                                    contentType: "application/pdf",
+                                    servletName: "svgtopdf.do"
+                                };
+                                cbio.download.initDownload(final_pdf_str, downloadOptions);
+
+                            });
+                            $("#cc_plots_data_download").click(function() {
```

Comment:
All similar jquery events need to be `unbind` first. After switching RNA version several times, if I click to download DATA, I got multiple files.